tests/components/lamarzocco/__init__.py

```diff
@@ -22,3 +25,24 @@ async def async_init_integration(
     """Set up the La Marzocco integration for testing."""
     await hass.config_entries.async_setup(mock_config_entry.entry_id)
     await hass.async_block_till_done()
+
+
+def get_bluetooth_service_info(
+    model: LaMarzoccoModel, serial: str
+) -> BluetoothServiceInfo:
+    """Return a mocked BluetoothServiceInfo."""
+    if model in (LaMarzoccoModel.GS3_AV, LaMarzoccoModel.GS3_MP):
+        name = f"GS3_{serial}"
+    elif model == LaMarzoccoModel.LINEA_MINI:
+        name = f"MINI_{serial}"
+    elif model == LaMarzoccoModel.LINEA_MICRA:
+        name = f"MICRA_{serial}"
```

Comment:
This could be replaced with a constant `dict` lookup

Reply:
done